src/xrCore/LocatorAPI.cpp

```diff
@@ -462,6 +459,7 @@ void CLocatorAPI::archive::open()
     if (hSrcFile)
         return;
 
+    while (char* sep = strchr((char *)*path, '\\')) *sep = '/';
```

Comment:
Please consider moving this code (slashes conversion) to separate function (named like ConvertGamePathToPlatformSpecificPath) with platform-specific implementation. It makes code more clear and allows to reduce code duplication.